account_invoice_inter_company/views/res_config_settings_view.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="base_setup.res_config_settings_view_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//div[@id='multi_company']//field[@name='module_inter_company_rules']/../../div[2]" position="before">
```

Comment:
I tried to install this module locally and this view didn't show up in the configuration. But in Runbot is shown! I investigated a bit and the cause is that I locally had the module `remove_odoo_enterprise` installed (see https://github.com/OCA/server-brand/pull/12).

Not sure whether module `remove_odoo_enterprise` works the correct way. However, is it possible to make here the inheritance of this view not depending on field `module_inter_company_rules`? This way we avoid this kind of conflicts, due to the fact that the configuration depends on a field (`module_inter_company_rules`) that is directly connected with enterprise apps.

Reply:
Thanks to notice this issue. I'll investigate

Reply:
@astirpe I tried to make modules compatible but I failed. I documented that in readme.
I think it's OK for you now. Thanks